### DISCUSSION TITLE
How can i add  action ,,enter" for    Form.Control type="password" ?

### DISCUSSION THREAD

**Discussion by virusek392** [twbs/bootstrap#38468] [General]
How can i add  action ,,enter" for    Form.Control type="password" ?

**Comment by patkon** [comment]
I'm sorry, I do not understand the question. What exactly do you want to achieve? Submit a form without a submit button?

**Comment by thisisnotdevin** [comment]
Im confused as well but if you mean like press enter to submit on the form after typing the password here is how:

```
<Form.Control
        type="password"
        value={password}
        onChange={(event) => setPassword(event.target.value)}
        onKeyDown={handleKeyDown}
      />
```
      
With this callback function:
```
    const handleKeyDown = (event) => {
    if (event.key === "Enter") {
      // Handle "Enter" key press
      ( submit your form)
    }
  };
```
